lib/apachetest.pm

```diff
@@ -23,6 +28,20 @@ our @EXPORT = qw(setup_apache2 setup_pgsqldb destroy_pgsqldb test_pgsql test_mys
 
 # Setup apache2 service in different mode: SSL, NSS, NSSFIPS, PHP7
 # Example: setup_apache2(mode => 'SSL');
+
+=head2 setup_apache2
+
+ setup_apache2();
+
+Setup Apache service in different mode: SSL, NSS, NSSFIPS, PHP7
+
+=over
+
+=item Example: setup_apache2(mode => 'SSL');
```

Comment:
There is no reason for this to be a list.
Also the code should be in a code block (eg one empty line above and below and one space prefix).